library/core/src/intrinsics/mod.rs

```diff
@@ -2294,10 +2294,11 @@ pub fn round_ties_even_f16(x: f16) -> f16;
 /// [`f32::round_ties_even`](../../std/primitive.f32.html#method.round_ties_even)
 #[rustc_intrinsic]
 #[rustc_nounwind]
-pub fn round_ties_even_f32(x: f32) -> f32;
+pub const fn round_ties_even_f32(x: f32) -> f32;
 
 /// Provided for compatibility with stdarch. DO NOT USE.
 #[inline(always)]
+#[rustc_allow_const_fn_unstable(core_intrinsics)]
 pub unsafe fn rintf32(x: f32) -> f32 {
```

Comment:
I think you meant to mark this as `const` as well?

Reply:
I don't think the `rintf` functions need to be constified at all.